docs/howtos/development.md

```diff
@@ -11,53 +29,53 @@ There are many options on how to get a local cluster for development. Here are a
 - [kind](https://github.com/kubernetes-sigs/kind)
 - [minikube](https://minikube.sigs.k8s.io/docs/start/)
 
-Assuming you have `k3d` installed, you can create a cluster with this command:
+Assuming you have `k3d` installed, you can create a cluster with this `make` target :
 
 ```
-k3d cluster create epinio
+make acceptance-cluster-setup
 ```
 
-This command should automatically update your default kubeconfig to point to
-the new cluster but if you need to save your kubeconfig manually you can do it with:
+(as the name also implies, this target is also used to prepare a cluster for the acceptance test suite in CI)
+
+This command writes the kubeconfig file to talk to the cluster in `tmp/acceptance-kubeconfig`.
+For the next steps to work, `KUBECONFIG` needs to be exported as so:
+
 
 ```
-k3d kubeconfig get epinio > epinio_kubeconfig
+export KUBECONFIG=$PWD/tmp/acceptance-kubeconfig
```

Comment:
I would suggest also to use the `k3d kubeconfig merge -ad` command, that will merge the current k3d configuration into the default kubeconfig file, so the cluster will be the correct one also accessing it from other shells, and without exporting the variable every time.

Reply:
Addressed on 7e80e84a